### DISCUSSION TITLE
Clock timer in live count

### DISCUSSION THREAD

**Discussion by mani2care** [grahampugh/erase-install#78] [Ideas]
can you use the timer in the live countdown as defined and also add the final window before reboot and ask the reboot 

Referance: 
https://babodee.wordpress.com/2021/03/30/handling-major-upgrades-and-minor-updates-for-macos-with-jamf/

**Comment by grahampugh** [comment]
There is already a confirmation option, `--confirm`, which will bring up a dialog before `startosinstall` runs. But with `startosinstall` there is no final option to reboot after the preparation has been made. That's different to Babodee's script which is using the Software Update GUI.
